Source/Methods/RFmxDPD/RFmxDPD.cs

```diff
@@ -131,13 +131,14 @@ public static LookupTableResults PerformLookupTable(RFmxSpecAnMX specAn, NIRfsg
                 PostDpdWaveform = referenceWaveform,
             };
             lutResults.PostDpdWaveform.WaveformName = referenceWaveform.WaveformName + "postLutDpd";
-            RFmxSpecAnMXDpdApplyDpdIdleDurationPresent idlePresent = referenceWaveform.IdleDurationPresent ? RFmxSpecAnMXDpdApplyDpdIdleDurationPresent.True : RFmxSpecAnMXDpdApplyDpdIdleDurationPresent.False;
+            lutResults.PostDpdWaveform.Script = lutResults.PostDpdWaveform.Script.Replace(referenceWaveform.WaveformName, lutResults.PostDpdWaveform.WaveformName);
```

Comment:
Sean, @manderseck's proposal was that we have a check to see if the replace operation fails, in case for some reason the input script does not match the current waveform. In this case we should call Configure Continuous Generation.

Reply:
when would that scenario happen? ApplyWaveformAttributes will call ConfigureContinuousGeneration if there isn't a script in the waveform.

Reply:
Sean and I decided that this was not necessary after all; we could not think of any use cases in which the two would be out of sync and the user would not be able to trace the problem.